ReText/tabactiontypes.py

```diff
@@ -0,0 +1,18 @@
+from enum import StrEnum
+
+from PyQt6.QtCore import QObject
+
+class TabActionTypes(StrEnum):
+
+    Unknown = ''
+
+    CopyFileName = QObject.tr('Copy File Name')
+    CopyFilePath = QObject.tr('Copy file path')
```

Comment:
On line 9 you capitalized all words, and here only one.

I think other interface capitalizes only the first word, can we make everything like that?